src/appkit.cs

```diff
@@ -16942,7 +16947,7 @@ public partial interface NSTextInputContext {
 		NSTextInputContext CurrentInputContext { get; }
 
 		[Export ("client")]
-		NSTextInputClient Client { get; }
+		INSTextInputClient Client { get; }
```

Comment:
They are new (so no break).

It looks like they weren't marked as 10,12 in the header file and were copied over without care. Fixing.

@dalexsoto This was to fix - warning BI1110: bmac: The property AppKit.NSTextInputContext.Client exposes a model (AppKit.NSTextInputClient). Please expose the corresponding protocol type instead (AppKit.INSTextInputClient).